helpers/lib/fonts.js

```diff
@@ -53,13 +54,19 @@ const fontProviders = {
             return collection;
         },
 
-        buildLink: function(fonts, fontDisplay) {
+        buildLink: function (fonts, fontDisplay) {
             const displayTypes = ['auto', 'block', 'swap', 'fallback', 'optional'];
             fontDisplay = displayTypes.includes(fontDisplay) ? fontDisplay : 'swap';
-            return `<link href="https://fonts.googleapis.com/css?family=${fonts.join('|')}&display=${fontDisplay}" rel="stylesheet">`;
+            const uri = `https://fonts.googleapis.com/css?family=${fonts.join('|')}&display=${fontDisplay}`
+            try {
+                const url = URL.parse(uri);
+                return `<link href="${url.format()}" rel="stylesheet">`;
+            } catch (error) {
+                throw new Error(`Invalid URL [${uri}]. Check configured fonts.`)
```

Comment:
Do you have examples of invalid urls that occurred? Maybe we can cover this case with unit tests

Reply:
yeah. I am not aware of any case of invalid URLs yet. Maybe because we had no way to know that given that only the browser was actually doing something with those URLs. Will add coverage to this code path.